cmd/api/src/api/bloodhoundgraph/conversions.go

```diff
@@ -31,6 +32,7 @@ const (
 func NodeToBloodHoundGraph(node *graph.Node) BloodHoundGraphNode {
 	var (
 		nodeKindLabel       = analysis.GetNodeKindDisplayLabel(node)
+		name, _             = node.Properties.GetWithFallback(common.Name.String(), "NO NAME OR ID", common.ObjectID.String()).String()
```

Comment:
If we're making these changes, we should probably line up the behavior between the two. Right now, the search endpoint has an additional case this one doesn't. Otherwise, I like this solution a lot

Reply:
Yes I intentionally kept it the same as it previously was to limit the change scope, but I'm all for consistency if we want to add display name as another fallback!